packages/react-label/src/components/Label/useLabelStyles.ts

```diff
@@ -2,25 +2,62 @@ import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
 import { LabelState } from './Label.types';
 
 /**
- * Styles for the root slot
+ * Styles for the label
  */
 const useStyles = makeStyles({
   root: theme => ({
-    // TODO Add default styles for the root element
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.base[300],
+    color: theme.alias.color.neutral.neutralForeground1,
+    display: 'inline-flex',
+    alignItems: 'flex-end',
   }),
 
-  // TODO add additional classes for different states and/or slots
+  disabled: theme => ({
+    color: theme.alias.color.neutral.neutralForegroundDisabled,
+  }),
+
+  required: theme => ({
+    color: theme.alias.color.red.foreground3,
+    fontSize: theme.global.type.fontSizes.base[300],
```

Comment:
Specifying fontSize here will prevent the asterisk from inheriting the font size from the parent label, so it could be the wrong size at small and large. Probably best to remove this and let it inherit the size.